kubernetes/loculus/templates/_common-metadata.tpl

```diff
@@ -239,8 +239,8 @@ organisms:
       {{ end }}
       {{ .website | toYaml | nindent 6 }}
       {{- end }}
-    referenceGenomes:
```

Comment:
The website changes are just for visualization purposes and should be reverted before merging. Currently the website cannot handle the unmerged reference genomes config.

Reply:
the integration test failures are due to this change actually (the mergedReferenceGenomes object does not contain the insdc ReferenceAccessions